eng/testing/linker/project.csproj.template

```diff
@@ -28,21 +28,11 @@
 
   <!-- Logic to override the default IlLink tasks that come from the SDK and use the one
   we use in dotnet/runtime repo -->
-
-  <PropertyGroup>
-    <_ILLinkTasksDirectoryRoot>$(PkgMicrosoft_NET_ILLink_Tasks)\tools\</_ILLinkTasksDirectoryRoot>
```

Comment:
The whole point of having this was in order to be able to override the tasks assembly to be able to pick a newer version instead, but I don't think that will be required any longer, so theoretically we could just remove the PackageReference from this project and let the SDK reference the version specified there right? @eerhardt is there still any value of referencing a newer version of the ILLink tasks than the one with the SDK?

Reply:
mm nvm looks like this package is also getting us the new linker so we might still want to keep the package reference.

Reply:
> The whole point of having this was in order to be able to override the tasks assembly to be able to pick a newer version instead

This is still happening. The IL.SDK's props file now overrides the property that points to the task assembly: https://github.com/mono/linker/blob/422f2f710707c1cf95dd513022c8fbef19f177dc/src/ILLink.Tasks/Sdk/Sdk.props#L15-L16